### PR TITLE
Chan

### PR DESCRIPTION
**Pull request by chanpham97**
merged w NickLiu and master
pls approve/yell at me

### CODE REVIEW

File: public/js/create.js [hunk 1]
```diff
@@ -0,0 +1,35 @@
+"use strict";
+
+//source: http://bootsnipp.com/snippets/featured/dynamic-form-fields-add-amp-remove-bs3
+//edit: http://formvalidation.io/examples/adding-dynamic-field/
+$(function()
+{
+	console.log("ooh ahh")
```
Comment (Howon) [inline public/js/create.js]: please

File: script/db/index.js [hunk 2]
```diff
@@ -196,4 +196,5 @@ module.exports = firebase => {
     }
 
   };
+>>>>>>> 80c474ac0fc72d75fb9ba524846065ebe8c84493
```
Comment (Howon) [inline script/db/index.js]: ???

File: public/js/create.js [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+  $(document).on("click", ".btn-add", function(e) {
+    e.preventDefault();
+    var controlForm = $(".controls form:first");
+    var currentEntry = $(this).parents(".entry:first");
+    var newEntry = $(currentEntry.clone()).appendTo(controlForm);
+    newEntry.find("input").val("");
+    controlForm.find(".entry:not(:last) .btn-add")
+      .removeClass("btn-add").addClass("btn-remove")
+      .removeClass("btn-success").addClass("btn-danger")
+      .html('<span class="glyphicon glyphicon-minus"></span>');
```
Comment (Howon) [inline public/js/create.js]: did you even run jshint

Comment (chanpham97) [inline public/js/create.js]: if i use double quotes in the html then the double quotes in the span act funny and it doesn't work :(

Comment (Howon) [inline public/js/create.js]: `<span class="glyphicon ... " ...</span>`

File: routes/index.js [hunk 4]
```diff
@@ -0,0 +1,74 @@
+"use strict";
```
Comment (francesar) [inline routes/index.js]: Remove this

**Comment by alanhdu** [comment]
Can we start squashing and rebasing the PRs? The logic's super hard to follow through all the commits.
